docs/js/contribution.md

```diff
@@ -0,0 +1,148 @@
+---
+id: contribution-model-sap-cloud-sdk-for-javascript-typescript
+title: Contribution model
+sidebar_label: Contribution model
+description:
+keywords:
+- sap
+- cloud
+- sdk
+- cloud native
+- cloud sdk
+- sap cloud sdk
+---
+
+## Introduction
+
+SAP Cloud SDK for JavaScript welcomes contributions that bring value to SDK users by simplifying and/or improving cloud application development within the SAP ecosystem. The contributed feature or API library becomes a part of SDK and is released under the SAP Cloud SDK umbrella into `@sap-cloud-sdk` to the [npmjs.com](https://www.npmjs.com/).
```

Comment:
I am still pretty sure, that this is generally not recognized as a contribution and I think it will cause confusion for people who would like to make actual contributions. Yes, you could use the overarching term of contribution, but this is rather an extension and to make sure there is no confusion, that's what I would call it.
There should be an obvious difference between our contributing guidelines and extension guidelines, because this is a completely different game.

Reply:
That's all good, @marikaner. What exactly do you want me to do considering I'm indifferent about naming?